model/component.js

```diff
@@ -65,34 +55,14 @@ class Component extends ParchKey {
     layer;
 
     /**
-     * The location of this component in the Architecture.
-     *
-     * @since 1.0.0
-     * @access public
-     *
-     * @type {object}
-     */
-    location;
-
-    /**
-     * How deep the component should be.
+     * The Component Features to be placed on the Architecture.
      *
      * @since 1.0.0
      * @access public
      *
-     * @type {number}
+     * @type {Array}
```

Comment:
This should actually be just a single component feature, since each component can only have 0 or 1 component features